controllers/likes.js

```diff
@@ -0,0 +1,130 @@
+import models from '../models';
+
+const { articles, likes, users } = models;
+
+class Like {
+  static async likeArticle(req, res) {
+    try {
+      const { articleSlug } = req.params;
+      const article = await articles.findOne({
+        where: { slug: articleSlug },
+        include: [{ model: likes }]
+      });
+
+      if (article === null) {
+        return res.status(404).json({
+          error: 'failed to find an article to like'
+        });
+      }
+      const user = await users.findOne({
+        where: { email: req.decoded.email },
+        include: [{ model: likes }]
+      });
+      const like = await likes.findOne({ where: { articleSlug: article.slug, userId: user.id }, attributes: ['typeState'] });
+
+      switch (true) {
```

Comment:
the switch `statement` and the `true`. am not understanding the use  of a switch here. On what or which condition do are you using this.